modules/apps/store/extensions/assets/webapp/pages/asset.jag

```diff
@@ -202,7 +217,13 @@ require('/modules/store.js').exec(function (ctx) {
     }
 
     function isStoreAdmin(){
-        return hasUserRole("Internal/store-admin");
+        if(authenticator.getLoggedInUser()) {
```

Comment:
authenticator.getLoggedInUser() has called in few places. Assign it into a variable and re-use.